src/test/java/org/cactoos/iterator/CycledTest.java

```diff
@@ -40,7 +40,7 @@
 public final class CycledTest {
 
     @Test
-    public void repeatIteratorTest() throws Exception {
```

Comment:
@ccostin93 Several `throws Exception` were removed but this does not seem to be part of the #1420 
@victornoel Do we need to correct?

Reply:
@marceloamadeu @ccostin93 good point but I think this is ok if the `throws` was not needed

Reply:
@marceloamadeu I removed the throws since they were not needed.